react/features/connection-indicator/statsEmitter.js

```diff
@@ -97,6 +105,24 @@ const statsEmitter = {
         });
     },
 
+    /**
+     * Emits an update to the listeners for end-to-end RTT updates.
+     *
+     * @param {JitsiParticipant} participant - The participant.
+     * @param {number} e2eRtt - The RTT.
+     * @returns {void}
+     * @private
+     */
+    _emitE2eRttUpdate(participant: Object, e2eRtt: number = -1) {
+        const id = participant.getId();
+        const region = participant.getProperty('region');
+        const callbacks = subscribers[id] || [];
+
+        callbacks.forEach(callback => {
+            callback(null /* stats */, e2eRtt, region);
```

Comment:
Not sure if you saw my question because it's now nested under and outdated changeset so I'll repost: Would it be difficult to take it further and make the callback itself be able to handle the stats object having e2eRtt and region instead of accessing them as separate arguments? I think there's already some logic in the callback that handles stats not having everything that's needed.